jenkins/src/cgcloud/jenkins/toil_jenkins_slave.py

```diff
@@ -287,7 +288,7 @@ def __configure_slurm( self ):
             JobCompType=jobcomp/none
 
             # ACCOUNTING
-            AccountingStorageLoc=/var/run/slurm-llnl/slurm-acct.txt
+            AccountingStorageLoc=%s
```

Comment:
With `heredoc()`, you can just say {slurm_acct_file} because it interpolates local variables.